docs/reference/release-notes/8.19.0.asciidoc

```diff
@@ -539,4 +539,17 @@ Snapshot/Restore::
 * Upgrade AWS Java SDK to 2.31.78 {es-pull}131050[#131050]
 * Upgrade AWS SDK to v1.12.746 {es-pull}122431[#122431]
 
+[discrete]
+[[known-issues-8.19.0]]
+=== Known issues
```

Comment:
I think we can revert this PR, because the issue doesn't occur in 8.19. By accident I forgot the remove the feature flag that controlled whether the merge optimization was enabled. This has been fixed: #128459